src/imp.rs

```diff
@@ -282,8 +282,24 @@ fn strike_through_attributes(
     strike_attrs: &mut Vec<Attribute>,
     ret: &mut TokenStream,
 ) {
+    let mut nostrike = false;
+
     dec_attrs.retain(|attr| {
-        if matches!(&attr.path[..], [TokenTree::Ident(kw)] if kw == "strikethrough") {
+        if matches!(&attr.path[..], [TokenTree::Ident(kw)] if kw == "nostrike") {
+            match &attr.value {
+                AttributeValue::Empty => {
+                    nostrike = true;
```

Comment:
For the subtrees, I think it's possible to do `strike_attrs.drain(..)` here. I haven't thought about whether that's wise.